src/static/helperTemplates/shopperCustomer.ts.hbs

```diff
@@ -1,40 +0,0 @@
-import { ShopperToken, stripBearer, ResponseError, getObjectFromResponse } from "@commerce-apps/core"
```

Comment:
Removed this helper function for retrieving an access token using the deprecated Shopper Customers `authorizeCustomer` endpoint. Now that the endpoint is removed, we can remove this helper function. This shouldn't be breaking as this is no longer supported by the API and all customers have migrated to SLAS